server/fileupdater.mjs

```diff
@@ -434,3 +435,12 @@ function v12HandDropShadow(properties) {
     properties.dropShadow = true;
   }
 }
+
+function v13fixedParentDefaults(properties) {
+  if (properties.type == 'card' || properties.type == 'pile' || properties.type == null){
+    if (properties.fixedParent=true)
+      properties.fixedParent=true;
+  } else if (properties.fixedParent=false){
```

Comment:
Need to check if fixedParent is not set, not if it is set to false.

Reply:
Done.